CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+
```

Comment:
I assume you were just triggering CI with this change and want to remove this commit from the PR

Reply:
yep that came in from some testing I had in master and was confusing me quite a bit :D